api/app/models/resource.rb

```diff
@@ -124,7 +124,7 @@ class Resource < ApplicationRecord
   before_validation :update_kind, :set_fingerprint!
   before_update :reset_stale_fields
   after_commit :queue_fetch_thumbnail, on: [:create, :update]
-  after_create :resource_to_event
+  after_commit :trigger_event_creation, on: [:create]
```

Comment:
I think I _did_ see a spec for this. Confirm—if not, add one.

Reply:
✔